---
title: 'scGIR: deciphering cellular heterogeneity via gene ranking in single-cell
  weighted gene correlation networks'
authors:
- F. Xu
- H. Hu
- H. Lin
- J. Lu
- F. Cheng
- J. Zhang
- X. Li
- J. Shuai
date: '2024-01-01'
publishDate: '2024-11-30T10:18:37.413290Z'
publication_types:
- article-journal
publication: '*Brief Bioinform*'
doi: 10.1093/bib/bbae091
abstract: Single-cell RNA sequencing (scRNA-seq) has emerged as a powerful tool for
  investigating cellular heterogeneity through high-throughput analysis of individual
  cells. Nevertheless, challenges arise from prevalent sequencing dropout events and
  noise effects, impacting subsequent analyses. Here, we introduce a novel algorithm,
  Single-cell Gene Importance Ranking (scGIR), which utilizes a single-cell gene correlation
  network to evaluate gene importance. The algorithm transforms single-cell sequencing
  data into a robust gene correlation network through statistical independence, with
  correlation edges weighted by gene expression levels. We then constructed a random
  walk model on the resulting weighted gene correlation network to rank the importance
  of genes. Our analysis of gene importance using PageRank algorithm across nine authentic
  scRNA-seq datasets indicates that scGIR can effectively surmount technical noise,
  enabling the identification of cell types and inference of developmental trajectories.
  We demonstrated that the edges of gene correlation, weighted by expression, play
  a critical role in enhancing the algorithm's performance. Our findings emphasize
  that scGIR outperforms in enhancing the clustering of cell subtypes, reverse identifying
  differentially expressed marker genes, and uncovering genes with potential differential
  importance. Overall, we proposed a promising method capable of extracting more information
  from single-cell RNA sequencing datasets, potentially shedding new lights on cellular
  processes and disease mechanisms.
links:
- name: URL
  url: https://www.ncbi.nlm.nih.gov/pubmed/38487851
---
